Assets/Scripts/BlockBehaviour.cs

```diff
@@ -18,17 +18,26 @@ public class BlockBehaviour : MonoBehaviour
     private bool _isPlayerStandingOn;
     private BlockFace _lastClickedFace;
     private BlockFaceBehaviour _blockFaceBehaviour;
+    private Renderer _renderer;
+    private bool _isFading = false;

```

Comment:
Not too comfortable having this in `BlockBehaviour`. Not all blocks will be "wall" blocks. Having_renderer is fine though.